js/comments.js

```diff
@@ -0,0 +1,70 @@
+const COMMENTS_PACK_AMOUNT = 5;
+
+const commentsCount = document.querySelector('.social__comment-count');
+const commentsLoader = document.querySelector('.comments-loader');
+const commentsList = document.querySelector('.social__comments');
+const commentTemplate = commentsList.querySelector('.social__comment');
+
+let shownComments = 0;
+
+const renderPack = (items, list, create) => {
+  const fragment = document.createDocumentFragment();
+
+  items.forEach((item) => {
+    const comment = create(item);
+    fragment.append(comment);
+  });
+
+  list.append(fragment);
+};
+
+const createComment = ({avatar, message, name}) => {
+  const commentElement = commentTemplate.cloneNode(true);
+  const commentAvatar = commentElement.querySelector('.social__picture');
+  const commentText = commentElement.querySelector('.social__text');
+
+  commentAvatar.src = avatar;
+  commentAvatar.alt = name;
+  commentText.textContent = message;
+
+  return commentElement;
+};
+
+const onLoadMoreClick = (comments) => {
```

Comment:
3. Но обработчик событие принимает только событие, поэтому нам нужно на верхнем уровне обозначить что такая функция будет

```suggestion
let onLoadMoreClick = null;
const createSliceRenderer = (comments)  = () => {
```

Reply:
Да, я понимала, что как0то нужно по ссылке обработчик передавать, чтобы потом его отписать. Но не смогла сообразить, как это)
Меня смущает конструкция `(comments)  = () => {` . Это опечатка, наверное - должно быть так `(comments) => {` ?

Reply:
Писал на скорую руку. Поправил. Тут смысл в том, что функция возвращает функцию